### PR TITLE
create stars

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
- [DEMO LINK](https://BilokonYaroslav.github.io/layout_stars/)
- [TEST REPORT LINK](https://BilokonYaroslav.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,49 @@
...
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
```
Comment (Virass) [inline src/index.html:25]: Change logic for set active stars amount. Set active stars use only second class for parrent, like stars--1... stars--3... etc...
And write css like 
`.stars--1 .stars__star:nth-child(-n+1),
	.stars--2 .stars__star:nth-child(-n+2),
	... continue this logic
	`

**Review by Virass — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
```
Comment (mykhalenych) [inline src/style.css:22]:
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
... {
  background: url(images/star-active.svg) no-repeat center;
  }
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+.stars {
+  height: 16px;
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
```
Comment (mykhalenych) [inline src/style.css:19]: last star shouldn't has margin-right

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.counteiner {
+  display: inline-block;
```
Comment (mykhalenych) [inline src/style.css:7]: it is redundant

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (BilokonYaroslav) [reply src/style.css:19]: without without margin-right the test does not pass

margin-right
margin-right2

Comment (VitaliyBondarenko1982) [reply src/style.css:15]: Not fixed.
Remove div with class counteiner and set last star margin-right: 0.
If you will do both fixes tests should pass

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (polosanya) [inline src/style.css:4]: Don't reset margins and paddings using (*) selector. It applies to all elements, which is not good for perfomance.
Better specify the exact elements

**Review by polosanya — APPROVED** [review]
Good job! I'm approving it, but please pay attention to my comment)

**Pull request closed without merging**